wpilibc/src/main/native/include/frc2/Timer.h

```diff
@@ -75,7 +75,8 @@ class Timer {
    * Start the timer running.
    *
    * Just set the running flag to true indicating that all time requests should
-   * be relative to the system clock.
+   * be relative to the system clock. Note that this method is a no-op if the
+   * timer is already running.
```

Comment:
Was this comment supposed to go against Reset()?

Reply:
No, it just clarifies that calling `Start()` on a running timer won't do anything.